vkcs/vpnaas/resource_site_connection.go

```diff
@@ -161,12 +165,51 @@ func ResourceSiteConnection() *schema.Resource {
 				ValidateDiagFunc: networking.ValidateSDN(),
 				Description:      "SDN to use for this resource. Must be one of following: \"neutron\", \"sprut\". Default value is project's default SDN.",
 			},
+			"traffic_selector_ep_merge": {
+				Type:         schema.TypeBool,
+				Optional:     true,
+				RequiredWith: []string{"sdn"},
+				Description:  "This argument controls whether multiple traffic selection rules for a network interface should be merged into one rule, or applied independently. When set to \"true\", this parameter can improve system performance, while providing more granular control over traffic selection when set to \"false\". __note__ Available only in sprut SDN.",
```

Comment:
'network interface' - this means something special in vkcs tf provider world. Here 'network interface' is a part of vpn world. Lets dig out what the network interface is and reword the description

Reply:
This argument controls whether multiple traffic selection rules for an IPSec site connection should be merged into one rule, or applied independently. When set to "true", this parameter results in single IPsec SA negotiation with multiple IP prefixes, while providing a separate negotiation that results in the multiple IPsec tunnels when set to "false".

Reply:
done